webui/react/src/pages/ExperimentList.tsx

```diff
@@ -79,6 +132,69 @@ const ExperimentList: React.FC = () => {
     handleFilterChange({ ...filters, username });
   }, [ filters, handleFilterChange ]);
 
+  const getBatchActions = useCallback((action: Action): Promise<void[] | Command> => {
```

Comment:
name it something other than get? maybe excute or send. this is actually sending the requests.